test/vmtests/vmtests/utils/imagecustomizer.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pathlib import Path
+
+from docker import DockerClient
+
+from .docker_utils import container_run
+
+
+# Run the containerized version of the imagecustomizer tool.
+def run_image_customizer(
```

Comment:
should we expose the partition extraction parameters too?

Reply:
Probably eventually. But might as well keep it simple for now.